### PR TITLE
Add employee id field to faculty_member

### PR DESCRIPTION
**Pull request by ayushjn20**

### CODE REVIEW

File: models/roles/faculty_member.py [hunk 1]
```diff
@@ -24,3 +32,11 @@ def __str__(self):
         designation = self.get_designation_display()
         department = self.department
         return f'{person} - {designation}, {department}'
+
+    def save(self, *args, **kwargs):
+        """
+        Override save method of FacultyMember class to automatically run validators of a model.
+        """
+
+        self.full_clean()
```
Comment (dhruvkb) [inline models/roles/faculty_member.py]: Make this validation opt-in using a keyword argument `run_validations`. Always running validations will lead to us not being able to make weird edge-case changes via the shell.

See https://github.com/IMGIITRoorkee/omniport-shell/pull/7#pullrequestreview-279956141 for example.

**Comment by dhruvkb** [comment]
@ayushjn20 or @algomaster99 please make sure faculties have, know and use this employee ID like students use their enrolment numbers. Only then is there any sense in adding another field. Otherwise there is a default `id` field on every model and faculties can continue to use their iitr.ac.in email addresses as their login IDs.

**Comment by ayushjn20** [comment]
They use their employee ID as username on acad.iitr.ac.in. Every other source of data for faculty information in the institute (like finance section, academic section, etc.) use the employee ID as the identifier for a faculty member, and it's the first thing they get when they are registered with the institution.

**Comment by dhruvkb** [comment]
Okay then, once the above review is resolved, I'll merge this PR.

**Pull request merged**